<launch>
  <include file="$(find mobile_launch)/launch/bringup_erratic_stack.launch" />
  <include file="$(find mobile_launch)/launch/vision.launch" />

  <node pkg="map_server" type="map_server" name="map_server" args="/home/tank/maps/walls.yaml"/>

  <include file="$(find erratic_navigation)/navigation_global/move_base.launch" />
  <include file="$(find erratic_navigation)/navigation_global/amcl_node.launch" />

  <node pkg="mobile_launch" type="delatcher.py" name="delatcher" />
  <node pkg="mobile_launch" type="tf_pose_pub.py" name="pose_pub" />
  <node pkg="mobile_launch" type="follower.py" name="follower">
    <param name="path_topic" value="/path"/>
    <param name="cmd_topic" value="/cmd_vel"/>
    <param name="num_times_spin" value="1"/>
    <param name="found_switch_topic" value="/switch_found"/>
    <param name="april_tag_frame1" value="switch_1"/>
    <param name="april_tag_frame2" value="switch_2"/>
  </node>
</launch>
